src/styles/Details.css

```diff
@@ -0,0 +1,26 @@
+.details-container,
+.details-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.details-container div {
+  display: flex;
+  padding: 0.5rem;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
+
+p {
+  font-size: 1.1rem;
+}
+
+.text-white {
+  color: #fff;
+}
+
+.info-text {
+  color: #fff;
+  font-size: 1.1rem;
+}
```

Comment:
- [ ] Your detail page design also need to follow the template design, as said in the requirement, you need to user the same layout, icones, paddings, margins, font-size, composition and space between elements.... so that your detail page design looks exactly the same as the template.

Your Design | the template
---|----
![image](https://github.com/Abdelaziz-Mahdi/Most-Gainers/assets/65068771/51c04245-c04e-4eec-b001-7e1b374ce959) |  ![image](https://github.com/Abdelaziz-Mahdi/Most-Gainers/assets/65068771/1e0d23cb-7c2f-4bbc-ba11-8686f2317fee)